app/pages/admin/organization-status-list.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import apiClient from 'panoptes-client/lib/api-client';
+import LoadingIndicator from '../../components/loading-indicator';
+import ProjectIcon from '../../components/project-icon';
+import Paginator from '../../talk/lib/paginator';
+
+class OrganizationStatusList extends Component {
+  constructor(props) {
+    super(props);
+    this.getOrganizations = this.getOrganizations.bind(this);
+    this.renderOrganizationList = this.renderOrganizationList.bind(this);
+    this.renderOrganizationListItem = this.renderOrganizationListItem.bind(this);
+    this.state = {
+      loading: false,
+      organizations: [],
+      error: null
+    };
+  }
+
+  componentWillMount() {
+    this.getOrganizations();
+  }
+
+  componentDidUpdate(prevProps) {
+    if (prevProps.location.query !== this.props.location.query) {
+      this.getOrganizations();
+    }
+  }
+
+  getOrganizations() {
+    const { query } = this.props.location;
+
+    const organizationsQuery = {
+      include: 'avatar'
+    };
+
+    const mergedQuery = Object.assign({}, organizationsQuery, query);
```

Comment:
this is more than necessary, but leaving for eventual sorting and filtering once functionality available from back-end.